src/interface/read_config_file.py

```diff
@@ -0,0 +1,81 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, file_config):
+        """ The Main function(constructor) -- creates the class attributes.
+
+        file_config -- string type, the function receives the file configuration XML file name.
+        self.xmld -- import data from the XML file.
```

Comment:
these values described are not parameters of the method they don't need to be described in the documentation of the method